main.py

```diff
@@ -1,52 +1,70 @@
 import uuid
 
 
-class Bank():
-    default_log='None'
-    def __init__(self,user_name,log=default_log):
-        self.user_name=user_name
-        self.log=str(log)
+class User():
+    def __init__(self,name):
+        self.name=name
         self.card=Card()
+
     def info(self):
-        print(f'User name:{self.user_name}')
-        return (self.card.info_card())
+        print(f'User name:{self.name}')
+        return (self.card.info())
```

Comment:
you don't need to do this call inside return - as your info() method just print info - so there is _no_ return value (void - oh does it calls in python)

Reply:
done.